validation/config.go

```diff
@@ -21,3 +35,7 @@ func init() {
 		panic(err)
 	}
 }
+
+func Config() config {
```

Comment:
exported func Config returns unexported type validation.config, which can be annoying to use